.kuzzlerc.sample

```diff
@@ -212,6 +212,10 @@
     // The profileIds applied to a user created with the API action
     // "security:createRestrictedUser"
     "restrictedProfileIds": ["default"],
+    // [supportCookieAuthentication]
+    // Enable Kuzzle secure support of Jwt in Cookie for the browser
+    // This option however is not compatible with [http.accessControlAllowOrigin] set to '*'
+    "supportCookieAuthentication": false,
```

Comment:
```suggestion
    // [supportCookieAuthentication]
    // Enables Kuzzle support for authentication tokens in cookies, for browsers
    // If activated, [http.accessControlAllowOrigin] must be set to a value different than '*'
    "supportCookieAuthentication": false,
```

Reply:
Also, I think this would be better to put that option in the HTTP section of the configuration, especially since there are interactions with other options there.